### PR TITLE
Add bids/asks methods, update README

### PR DESCRIPTION
**Pull request by gkaemmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,25 @@ All API methods that return lists (`getMarkets`, `getOrders`, `getOrderFills`, a
 
 All of these methods also take a optional `page` argument, which you can use to fetch additional pages.
 
+## Shares primer
+
+Veil markets each have two tokens: LONG and SHORT. By holding shares of LONG or SHORT tokens, you hold a "position" in a market. When a market ends, its LONG and SHORT shares are redeemable for ETH, with the rates depending on the market's result. 
+
+In yes/no markets (such as, "Will ETH be above $100 at the end of 2018?"), the payout goes entirely to one share token, LONG if the market resolves to "Yes", and SHORT if the market resolves to "No".
```
Comment (pfletcherhill) [inline README.md]: `In yes/no markets (e.g. "Will ETH be above $100 at the end of 2018?"), the payout goes entirely to one share token—LONG if the market resolves to "Yes" and SHORT if the market resolves to "No".`

File: README.md [hunk 2]
```diff
@@ -35,6 +35,25 @@ All API methods that return lists (`getMarkets`, `getOrders`, `getOrderFills`, a
...
+
+In scalar markets (such as, "What will be the price of ETH at the end of 2018?"), the payout is split between LONG and SHORT tokens according to where the result falls within the market's "bounds" (set by `minPrice` and `maxPrice`).
```
Comment (pfletcherhill) [inline README.md]: `In scalar markets (e.g. "What will be the price of ETH at the end of 2018?"), the payout is split between LONG and SHORT tokens according to where the result (e.g. the price of ETH) falls within the market's "bounds" (set by `minPrice` and `maxPrice`).`

File: README.md [hunk 3]
```diff
@@ -35,6 +35,25 @@ All API methods that return lists (`getMarkets`, `getOrders`, `getOrderFills`, a
...
+
+> Note: Together, 1 LONG share and 1 SHORT share are always redeemable for exactly 1 ETH.
+
+The price of a Veil share token is therefore always somewhere between 0 and 1 ETH/share, depending on what the market predicts that each token's payout will be.
```
Comment (pfletcherhill) [inline README.md]: `The price of a Veil share token is therefore always somewhere between 0 and 1 ETH per share, depending on what the market predicts that each token's payout will be.`

File: README.md [hunk 4]
```diff
@@ -35,6 +35,25 @@ All API methods that return lists (`getMarkets`, `getOrders`, `getOrderFills`, a
 
 All of these methods also take a optional `page` argument, which you can use to fetch additional pages.
 
+## Shares primer
```
Comment (pfletcherhill) [inline README.md]: Should mention Augur at least once here

File: README.md [hunk 5]
```diff
@@ -88,31 +107,23 @@ Fetches details about a single market. Example response:
 }
 ```
 
-### `veil.getOrders(market: Market, options?: { page: number })`
+### `veil.getBids(market: Market, tokenType: "long" | "short")` and `veil.getAsks(market: Market, tokenType: "long" | "short")`
 
-Fetches the open orders in a market. Example response:
+The `getBids` and `getAsks` methods let you fetch the orderbook for a market. You can fetch orders for either LONG or SHORT tokens by passing `tokenType` as a second argument (in Veil markets, the LONG and SHORT orderbooks are always mirror images of each other).
```
Comment (pfletcherhill) [inline README.md]: `order book` (two words)

File: README.md [hunk 6]
```diff
@@ -122,31 +133,19 @@ Fetches the open orders in a market. Example response:
...
-> 
-> A `longPrice` of 6000 on an order where `longSide` is `buy` means that the creator of this order is willing to buy long shares for 0.6 ETH/share *OR* sell short shares for 0.4 ETH/share.
-> 
-> For more information about this, ask a question on our [Discord](https://discord.gg/RcWDAr9)
-
-### `veil.getOrderFills(market: Market, options?: { page: number })`
+### `veil.getOrderFills(market: Market, tokenType: "long" | "short", options?: { page: number })`
 
-Fetches the order fill history in a market. Example response:
+Fetches the order fill history in a market for tokens of type `tokenType` (LONG or SHORT). Example response:
```
Comment (pfletcherhill) [inline README.md]: How are we doing categorical markets? Would we ever support Augur categorical markets?

File: README.md [hunk 7]
```diff
@@ -88,31 +107,23 @@ Fetches details about a single market. Example response:
...
-      token: "0x598b46d68e3e03f810a45d7d8dc9af5afdbafd56",
-      tokenAmount: "100000000000000",
-      "tokenAmountFilled":"0",
-      currency: "0xe7a67a41b4d41b60e0efb60363df163e3cb6278f",
-      currencyAmount: "229300000000000000",
-      currencyAmountFilled: "0",
-      postOnly: false,
-      market: null
+      price: "7000",
+      tokenAmount: "100000000000000"
```
Comment (pfletcherhill) [inline README.md]: Not worth doing something like this is it?
```
{ price: {raw: "7000", float: 0.7}, tokenAmount: {raw: "100000000000000", float: 1.0} }
```

File: README.md [hunk 8]
```diff
@@ -160,7 +159,7 @@ Fetches the order fill history in a market. Example response:
 
 Creates a Veil quote, which is used to calculate fees and generate an unsigned 0x order, which is required to create a Veil order.
 
-> **Note**: As above, `price` here is a number between 0 and `market.numTicks`. A price of 6000 is equivalent to 0.6 ETH/share.
+> **Note**: `price` is a number between 0 and `market.numTicks`, which is always 10000. A price of 6000 is equivalent to 0.6 ETH/share.
```
Comment (pfletcherhill) [inline README.md]: `which is always 10000 for Veil markets`

File: src/Veil.ts [hunk 9]
```diff
@@ -26,8 +26,8 @@ export interface Market {
 
 export interface Order {
   uid: string;
-  longPrice: string;
-  longSide: "buy" | "sell";
+  price: string;
+  side: "buy" | "sell";
```
Comment (pfletcherhill) [inline src/Veil.ts]: Does coinbase pro use `side` as this key?

**Comment by pfletcherhill** [comment]
Also todo:
* [ ] install info (i.e. `yarn add veil-js`)
* [ ] support info (who it's maintained by, our twitter, discord, email, etc)

File: README.md (hunk 7 above)
Comment (gkaemmer) [reply README.md]: Maybe worth eventually adding convenience fields like `tokenAmountFloat` to 1) keep it backward compatible and 2) allow people to ignore it

File: src/Veil.ts (hunk 9 above)
Comment (gkaemmer) [reply src/Veil.ts]: yeah

**Pull request merged**